### ISSUE TITLE
ios does not build in 6.0.1-rc.0

### ISSUE THREAD

**Issue by stefan-messmer** [react-native-community/react-native-svg#530]
In release candidate 6.0.1-rc.0, there seems to be a change how the dependencies PerformanceBezier & QuartzBookPack are downloaded. It results in a mostly empty folder for both sub projects with a lot of files missing.

**Comment by dantman** [comment]
Same, I get this error when I build a release from the command line with fastlane, but not when I test in debug or release mode in Xcode.

node_modules/react-native-svg/ios/Text/RNSVGTSpan.h:9:9: 'PerformanceBezier/PerformanceBezier.h' file not found

**Comment by dantman** [comment]
I tried manually running the postinstall script.

```
> react-native-svg@6.0.1-rc.0 postinstall {app}/node_modules/react-native-svg
> node scripts/install.js


 Start downloading PerformanceBezier to `{app}/node_modules/react-native-svg/ios/PerformanceBezier`
Download PerformanceBezier library from github failed with err: { Error: ENOTEMPTY: directory not empty, rename '{app}/node_modules/react-native-svg/1511979249249-20925124952102148/PerformanceBezier-master' -> '{app}/node_modules/react-native-svg/ios/PerformanceBezier'
  errno: -66,
  code: 'ENOTEMPTY',
  syscall: 'rename',
  path: '{app}/node_modules/react-native-svg/1511979249249-20925124952102148/PerformanceBezier-master',
  dest: '{app}/node_modules/react-native-svg/ios/PerformanceBezier' }
Download PerformanceBezier library success!

 Start downloading QuartzBookPack to `{app}/node_modules/react-native-svg/ios/QuartzBookPack`
Download QuartzBookPack library from github failed with err: { Error: ENOTEMPTY: directory not empty, rename '{app}/node_modules/react-native-svg/1511979250244-07748628871495566/QuartzBookPack-master' -> '{app}/node_modules/react-native-svg/ios/QuartzBookPack'
  errno: -66,
  code: 'ENOTEMPTY',
  syscall: 'rename',
  path: '{app}/node_modules/react-native-svg/1511979250244-07748628871495566/QuartzBookPack-master',
  dest: '{app}/node_modules/react-native-svg/ios/QuartzBookPack' }
Download QuartzBookPack library success!
```

**Comment by dantman** [comment]
This temporarily works around the issue:
```
$ cd node_modules/react-native-svg/
$ rm -rf ios/PerformanceBezier/ ios/QuartzBookPack/
$ npm run postinstall

> react-native-svg@6.0.1-rc.0 postinstall {app}/node_modules/react-native-svg
> node scripts/install.js


 Start downloading PerformanceBezier to `{app}/node_modules/react-native-svg/ios/PerformanceBezier`
Download PerformanceBezier library success!

 Start downloading QuartzBookPack to `{app}/node_modules/react-native-svg/ios/QuartzBookPack`
Download QuartzBookPack library success!
```

**Comment by turtec** [comment]
@dantman your workaround works for me

**Comment by hatlestad** [comment]
I experienced the same issue. I tried the workaround @dantman presented, which cleared the error presented, but then I was getting a duplicate error this time for QuartzBookPack

`'QuartzBookPack/QuartzBookPack.h' file not found` 

or something of that nature.

My temporary fix was to revert to 6.0.0 which resolved all of my issues.

**Comment by dustinsavery** [comment]
I'll try to look into this over the weekend and see if I can find a root cause. If any of you know what's causing this and can submit a PR, we can get an RC1 going quick

**Comment by ulisinn** [comment]
Got it to work, but I also had to remove tvOS

**Comment by tybro0103** [comment]
Ran into the same. The fix from @dantman worked for me, but like @ulisinn mentioned, I had to remove the tvOS lib as well. It gets added to "Link Binary with Libraries" build phase section by `react-native link`. [See here](https://github.com/oblador/react-native-vector-icons/issues/461#issuecomment-299258473) for more details.

**Comment by dustinsavery** [comment]
`react-native-svg@6.0.1-rc.1` has been released and it fixes this issue.

**Comment by thibault12** [comment]
I still get errors downloading PerformanceBezier and QuartzBookPack with react-native-svg@6.0.1-rc.1

**Comment by dustinsavery** [comment]
@thibault12 the same issues on a cleaned project? If that’s the case, i’ll re-open

**Comment by sampsasaarela** [comment]
@dustinsavery I also still get errors downloading `PerformanceBezier` and `QuartzBookPack` with `react-native-svg@6.0.1-rc.1`. The project is new so it is clean. I tried it multiple times.

**Comment by glacjay** [comment]
downgrade to 5.5.1 to temperarily fix this problem :(

**Comment by nishantxeno** [comment]
@glacjay Thanks, man. Saved me a day

**Comment by msand** [comment]
I've just released 6.0.1-rc.2 which removes PerformanceBezier and QuartzBookPack. Can you please test it? 
@stefan-messmer
@dantman
@turtec
@hatlestad
@ulisinn
@tybro0103
@thibault12
@sampsasaarela
@glacjay
@nishantxeno

**Comment by sampsasaarela** [comment]
Thanks, @msand, I can confirm it worked now.
